install/installer/pkg/components/ws-manager/deployment.go

```diff
@@ -112,6 +116,13 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil, err
 	}
 
+	if vol, mnt, _, ok := common.CustomCACertVolume(ctx); ok {
+		podSpec.Volumes = append(podSpec.Volumes, *vol)
+		pod := podSpec.Containers[0]
+		pod.VolumeMounts = append(pod.VolumeMounts, *mnt)
+		podSpec.Containers[0] = pod
```

Comment:
@sagor999 's suggestion.

```suggestion
		container := podSpec.Containers[0]
		container.VolumeMounts = append(container.VolumeMounts, *mnt)
		podSpec.Containers[0] = container
```